README.md

```diff
@@ -67,8 +70,55 @@ Run all tests:
 env ROLLBAR_TOKEN=<token> stack test
 ```
 
+## Contributors
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
 ## License
 
-[MIT](LICENSE)
+MIT, see [the LICENSE file](LICENSE).
+
+## Contributing
+
+Do you want to contribute to this project? Please take a look at our [contributing guideline](/docs/CONTRIBUTING.md) to know how you can help us build it.
+
+---
+<img src="https://cdn.stackbuilders.com/media/images/Sb-supports.original.png" alt="Stack Builders" width="50%"></img>
+[Check out our libraries](https://github.com/stackbuilders/) | [Join our team](https://www.stackbuilders.com/join-us/)
 
 [rollbar-api]: https://docs.rollbar.com/
+
+## Contributors ✨
```

Comment:
@DavidMazarro Thank you for including this section; I believe it is great to provide more visibility about the contributors. Just one minor thing: in terms of layout, I believe it is best to leave the "Contributing" section at the bottom because it contains a footer.